sanity-mz-2022/schemas/blog.js

```diff
@@ -0,0 +1,50 @@
+export default {
+  name: 'blog',
+  title: 'Blog',
+  type: 'document',
+  fields: [
+    {
+      name: 'image',
+      title: 'Image',
+      type: 'array',
+      of: [{ type: 'image'}],
+      options: {
+        hotspot: true
+      }
+    },
+    {
+      name: 'blogTitle',
```

Comment:
I wouldn't name these "BlogTitle" since you'll access them with `schemaName.fieldName` so it would become `blog.blogTitle` in code instead of something maybe more simple like `blog.title`
this applies to all the below ones as well